.travis.yml

```diff
@@ -1,9 +1,12 @@
 language: python
 python:
   - '2.7'
-before_install:
```

Comment:
OK. Was it just the all-access tests that were the problem? The travis PR build passed